### PR TITLE
[DOC] RDoc for Complex

### PR DESCRIPTION
**Pull request by BurdetteLamar**
#numerator and #denominator.

### CODE REVIEW

File: complex.c [hunk 1]
```diff
@@ -1492,21 +1498,28 @@ nucomp_denominator(VALUE self)
...
+ *   c.numerator                                # => (8+9i)
  *
- *    c = Complex('1/2+2/3i')  #=> ((1/2)+(2/3)*i)
- *    n = c.numerator          #=> (3+4i)
- *    d = c.denominator        #=> 6
- *    n / d                    #=> ((1/2)+(2/3)*i)
- *    Complex(Rational(n.real, d), Rational(n.imag, d))
- *                             #=> ((1/2)+(2/3)*i)
- * See denominator.
+ * Related: Complex.denominator.
```
Comment (peterzhu2118) [inline complex.c:1522]: Did you mean `Complex#denominator`?

```suggestion
 * Related: Complex#denominator.
```

File: complex.c [hunk 2]
```diff
@@ -1492,21 +1498,28 @@ nucomp_denominator(VALUE self)
...
+ *   c.real.denominator # => 3
+ *   c.imag.numerator   # => 3
+ *   c.imag.denominator # => 4
+ *   # Real and imaginary parts for new Complex c.numerator.
+ *   result_real = c.real.numerator * c.imag.denominator # => 8
+ *   result_imag = c.imag.numerator * c.real.denominator # => 9
+ *   # New Complex c.numerator (Q.E.D.).
+ *   result = Complex(result_real, result_imag) # => (8+9i)
+ *   c.numerator                                # => (8+9i)
  *
```
Comment (peterzhu2118) [inline complex.c:1521]: I find this example to be difficult to understand. Can we explain using words instead? I think the numerator is the Complex created from the numerators of the real and imaginary parts after converting them to the least common denominator.

File: complex.c (hunk 1 above)
Comment (BurdetteLamar) [reply complex.c]: Fixed.

File: complex.c (hunk 2 above)
Comment (BurdetteLamar) [reply complex.c]: Revised.

**Review by peterzhu2118 — APPROVED** [review]

**Pull request merged**